test/system/065-cp.bats

```diff
@@ -7,6 +7,312 @@
 
 load helpers
 
+@test "podman cp file from host to container" {
+    skip_if_remote "podman-remote does not yet handle cp"
+
+    srcdir=$PODMAN_TMPDIR/cp-test-file-host-to-ctr
+    mkdir -p $srcdir
+    # Create multiple files which prevents some (expensive) clean ups via
+    # podman-exec.
+    echo "This file is on the host" > $srcdir/hostfile
+    echo "This file is on the host (1)" > $srcdir/hostfile1
+    echo "This file is on the host (2)" > $srcdir/hostfile2
+
+    run_podman run -d --name cpcontainer --workdir=/srv $IMAGE sleep infinity
+
+    # Copy to root.
+    run_podman cp $srcdir/hostfile cpcontainer:/
+    run_podman exec cpcontainer cat /hostfile
+    is "$output" "This file is on the host"
+
+    run_podman cp $srcdir/hostfile cpcontainer:/anotherbase.txt
+    run_podman exec cpcontainer cat /anotherbase.txt
+    is "$output" "This file is on the host"
+
+    # Copy to a non-root directory (varied input syntax).
+    run_podman cp $srcdir/hostfile cpcontainer:/tmp
+    run_podman exec cpcontainer cat /tmp/hostfile
+    is "$output" "This file is on the host"
+
+    run_podman cp $srcdir/hostfile1 cpcontainer:/tmp/
+    run_podman exec cpcontainer cat /tmp/hostfile1
+    is "$output" "This file is on the host (1)"
+    run_podman exec cpcontainer rm /tmp/hostfile
```

Comment:
My suggested diff does not do this 'rm'. It wasn't clear to this reader what the purpose of that was.